pkgs/development/libraries/gdk-pixbuf/default.nix

```diff
@@ -73,21 +73,25 @@ stdenv.mkDerivation rec {
   '';
 
   postInstall =
-    # All except one utility seem to be only useful during building.
+    # meson erroneously installs loaders with .dylib extension on Darwin.
+    # Their @rpath has to be replaced before gdk-pixbuf-query-loaders looks at them.
+    stdenv.lib.optionalString stdenv.isDarwin ''
+      for f in $out/${passthru.moduleDir}/*.dylib; do
+          install_name_tool -change @rpath/libgdk_pixbuf-2.0.0.dylib $out/lib/libgdk_pixbuf-2.0.0.dylib $f
```

Comment:
No, it is a normal lib, not a GLib module, and it was `.dylib` in gdk-pixbuf-2.36.7.

Reply:
Oh, I thought the `install_name` needs to match the filename.

Reply:
I'm not sure that you got this right. The old libs looked like this:
```
/nix/store/bkw80djwwgi868v02nd9xhizbxbj6f22-gdk-pixbuf-2.36.7/lib
├── gdk-pixbuf-2.0
│   └── 2.10.0
│       ├── loaders
…
│       │   ├── libpixbufloader-png.la
│       │   ├── libpixbufloader-png.so
…
│       └── loaders.cache
├── libgdk_pixbuf-2.0.0.dylib
├── libgdk_pixbuf-2.0.dylib -> libgdk_pixbuf-2.0.0.dylib
…
```
and the new libs look like this:
```
/nix/store/xr59fg3v6js0kkpbvsdvfn8382fsqqz0-gdk-pixbuf-2.36.12/lib
├── gdk-pixbuf-2.0
│   └── 2.10.0
│       ├── loaders
…
│       │   ├── libpixbufloader-png.dylib (after this PR becomes .so)
…
│       └── loaders.cache
├── libgdk_pixbuf-2.0.0.dylib
├── libgdk_pixbuf-2.0.dylib -> libgdk_pixbuf-2.0.0.dylib
…
 `$out/lib/libgdk_pixbuf-2.0.0.dylib` is not renamed to `.so` because it is not in `$out/${passthru.moduleDir}` (i.e. `$out/lib/gdk-pixbuf-2.0/2.10.0/loaders`).

`install_name_tool` updates `libpixbufloader-png.dylib` et al. to load `$out/lib/libgdk_pixbuf-2.0.dylib`.

Reply:
Of course, you are right, I was misreading that.